src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/InterfaceMethodsShouldBeCallableByChildTypes.cs

```diff
@@ -66,16 +66,16 @@ private static bool ShouldExcludeOperationBlock(ImmutableArray<IOperation> opera
             {
                 IBlockOperation block = operationBlocks[0] as IBlockOperation;
 
-                // An operation block that's not even a block - don't analyze the error cases.
+                // Analyze IBlockOperation blocks.
+                // TODO: Handle expression bodied members https://github.com/dotnet/roslyn-analyzers/issues/1345
```

Comment:
I thought we had expression-bodied members be blocks as well, from the IOperation standpoint.